spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -347,4 +347,12 @@ private boolean executorCacheLocalityEnabledInternal() {
         .defaultValue(SparkSQLProperties.EXECUTOR_CACHE_LOCALITY_ENABLED_DEFAULT)
         .parse();
   }
+
+  public boolean enableColumnStats() {
```

Comment:
I think this would be good as an escape valve. Imagine we have a bug in our stat delivery but Spark does not. Do we want to force all datasources to ignore CBO to be compatible with Iceberg?